modules/structs/repo.go

```diff
@@ -42,17 +42,26 @@ type Repository struct {
 	// swagger:strfmt date-time
 	Created time.Time `json:"created_at"`
 	// swagger:strfmt date-time
-	Updated                   time.Time   `json:"updated_at"`
-	Permissions               *Permission `json:"permissions,omitempty"`
-	HasIssues                 bool        `json:"has_issues"`
-	HasWiki                   bool        `json:"has_wiki"`
-	HasPullRequests           bool        `json:"has_pull_requests"`
-	IgnoreWhitespaceConflicts bool        `json:"ignore_whitespace_conflicts"`
-	AllowMerge                bool        `json:"allow_merge_commits"`
-	AllowRebase               bool        `json:"allow_rebase"`
-	AllowRebaseMerge          bool        `json:"allow_rebase_explicit"`
-	AllowSquash               bool        `json:"allow_squash_merge"`
-	AvatarURL                 string      `json:"avatar_url"`
+	Updated                      time.Time   `json:"updated_at"`
+	Permissions                  *Permission `json:"permissions,omitempty"`
+	HasIssues                    bool        `json:"has_issues"`
+	ExternalTracker              bool        `json:"external_tracker"`
```

Comment:
I prefer use a struct or map to store all external tracker's informations.

Reply:
Maybe structs for InternalTracker, ExternalTracker and ExternalWiki (nil if not existing)?

Reply:
@davidsvantesson Yes, that's what I mean.

Reply:
I changed the reported structure of the repository, but kept the edit options in a flat structure to avoid complexity when entering options (as they are anyhow optional)